subiquity/cmd/tui.py

```diff
@@ -78,6 +78,9 @@ def parse_options(argv):
     parser.add_argument('--click', metavar="PAT", action=ClickAction,
                         help='Synthesize a click on a button matching PAT')
     parser.add_argument('--answers')
+    parser.add_argument('--source', default=[], action='append',
+                        dest='sources', metavar='URL',
+                        help='install from url instead default.')
```

Comment:
This is append mode which means the source value may be a list, not sure if the UI changes handled the case where we've passed --source=src1 --source=src2

Can you confirm ?

Reply:
the ui only shows 'Command Line Source'
the code does the right thing.

Reply:
"install from url instead default" -- word or two missing?

Reply:
adjusted to 'install from url instead of default'.